plugins/templates-maven-plugin/src/main/java/com/google/cloud/teleport/plugin/maven/TemplatesStageMojo.java

```diff
@@ -1038,6 +1119,11 @@ static String generateFlexTemplateImagePath(
                 + containerName);
   }
 
+  static String generateStagingFlexTemplateImagePath(
+      String containerName, String stagingArtifactRegistry, String stagePrefix) {
```

Comment:
Nit: this would probably read clearer if we took parameters in the same order that we use them below.

Reply:
that's fair. The current order was due to `generateFlexTemplateImagePath` had same parameter order (after removing parameters that are not used in this method)